docs/site/_includes/getting_started/vsphere/STEP_ENV_RU.md

```diff
@@ -74,102 +81,46 @@ govc tags.attach -c k8s-zone test_zone /DC/datastore/test_lun
-  VirtualMachine.Interact.DefragmentAllDisks VirtualMachine.Interact.DeviceConnection VirtualMachine.Interact.DisableSecondary \
-  VirtualMachine.Interact.DnD VirtualMachine.Interact.EnableSecondary VirtualMachine.Interact.GuestControl \
-  VirtualMachine.Interact.MakePrimary VirtualMachine.Interact.Pause VirtualMachine.Interact.PowerOff \
-  VirtualMachine.Interact.PowerOn VirtualMachine.Interact.PutUsbScanCodes VirtualMachine.Interact.Record \
-  VirtualMachine.Interact.Replay VirtualMachine.Interact.Reset VirtualMachine.Interact.SESparseMaintenance \
-  VirtualMachine.Interact.SetCDMedia VirtualMachine.Interact.SetFloppyMedia VirtualMachine.Interact.Suspend \
-  VirtualMachine.Interact.TerminateFaultTolerantVM VirtualMachine.Interact.ToolsInstall \
-  VirtualMachine.Interact.TurnOffFaultTolerance VirtualMachine.Inventory.Create \
-  VirtualMachine.Inventory.CreateFromExisting VirtualMachine.Inventory.Delete \
-  VirtualMachine.Inventory.Move VirtualMachine.Inventory.Register VirtualMachine.Inventory.Unregister \
-  VirtualMachine.Namespace.Event VirtualMachine.Namespace.EventNotify VirtualMachine.Namespace.Management \
-  VirtualMachine.Namespace.ModifyContent VirtualMachine.Namespace.Query VirtualMachine.Namespace.ReadContent \
-  VirtualMachine.Provisioning.Clone VirtualMachine.Provisioning.CloneTemplate VirtualMachine.Provisioning.CreateTemplateFromVM \
-  VirtualMachine.Provisioning.Customize VirtualMachine.Provisioning.DeployTemplate VirtualMachine.Provisioning.DiskRandomAccess \
-  VirtualMachine.Provisioning.DiskRandomRead VirtualMachine.Provisioning.FileRandomAccess VirtualMachine.Provisioning.GetVmFiles \
-  VirtualMachine.Provisioning.MarkAsTemplate VirtualMachine.Provisioning.MarkAsVM VirtualMachine.Provisioning.ModifyCustSpecs \
-  VirtualMachine.Provisioning.PromoteDisks VirtualMachine.Provisioning.PutVmFiles VirtualMachine.Provisioning.ReadCustSpecs \
-  VirtualMachine.State.CreateSnapshot VirtualMachine.State.RemoveSnapshot VirtualMachine.State.RenameSnapshot VirtualMachine.State.RevertToSnapshot \
-  Cns.Searchable StorageProfile.View
-
-govc permissions.set  -principal имя_пользователя -role kubernetes /DC
+govc role.create deckhouse \
+   Cns.Searchable Datastore.AllocateSpace Datastore.Browse Datastore.FileManagement \
+   Global.GlobalTag Global.SystemTag Network.Assign StorageProfile.View \
+   $(govc role.ls Admin | grep -F -e 'Folder.' -e 'InventoryService.' -e 'Resource.' -e 'VirtualMachine.')
 ```
-{% endsnippetcut %}
 
-### Сборка образа виртуальных машин
+Назначьте пользователю роль на объекте `vCenter`:
 
-1. [Установить Packer](https://learn.hashicorp.com/tutorials/packer/get-started-install-cli).
-1. Склонировать [репозиторий Deckhouse](https://github.com/deckhouse/deckhouse/):
-   {% snippetcut %}
-```bash
-git clone https://github.com/deckhouse/deckhouse/
+```shell
+govc permissions.set -principal <username>@vsphere.local -role deckhouse /
```

Comment:
```suggestion
govc permissions.set -principal <username>@vsphere.local -role deckhouse
```
typo ?

Reply:
No. This is the path to the root object (vCenter)